benchcab/data/test/integration.sh

```diff
@@ -39,6 +32,7 @@ fluxsite:
   experiment: AU-Tum
   pbs:
     storage:
+      - gdata/wd9
```

Comment:
This needs to be added to `render_job_script()` in `pbs.py` with the other storage flags. It isn't a user entry because benchcab would fail if it is missing. 

We need to change all these hard-coded data to be handled differently, but that's another issue.

Reply:
And it should be removed from `integration.sh`

Reply:
Do we assume access to `ks32` for `wd9` (can write it off as a comment)